pages/item/_itemId/index.vue

```diff
@@ -19,13 +23,15 @@
             />
             <h2
               v-if="loaded && item.OriginalTitle"
-              class="text-subtitle-1 text-truncate"
+              class="text-subtitle-1"
+              :class="{ 'text-center': !$vuetify.breakpoint.mdAndUp }"
```

Comment:
Why not move this class to the parent `v-col` that way we don't have to duplicate it several times

Reply:
Cause the parent `v-col` is also parent to the one L85 and this one doesn't need to be left aligned :)